packages/matter-node.js/test/IntegrationTest.ts

```diff
@@ -963,6 +962,7 @@ describe("Integration Test", () => {
             assert.deepEqual(pushedUpdates, [{ value: true, time: startTime + 2 * 1000 + 100 }]);
         });
 
+        /*
```

Comment:
Could include a note on why commented and/or when to uncomment

Reply:
agreed